fs/rails-base-graphql-api
# Make updateUser return user type

### Summary

Make updateUser mutation return User type

### How it works

![image](https://user-images.githubusercontent.com/57901245/83886650-09b06980-a750-11ea-919f-caf809c457af.png)

### Test plan

* Open [GraphiQL Interface](http://localhost:3000/graphiql)
* Make request using schema:
```graphql
mutation UpdateUser($firstName: String, $lastName: String) {
  updateUser(
    firstName: $firstName,
    lastName: $lastName,
    currentPassword: "password"
    password: "qwerty"
  ) {
    id
    email
    firstName
    lastName
  }
}
```
* User type should be returned.

### Review notes

While reviewing pull-request (especially when it's your pull-request),
please make sure that:

- you understand what problem is solved by PR and how is it solved
- new tests are in place, no redundant tests
- DB schema changes reflect new migrations
- newly introduced DB fields have indexes and constraints
- there are no missed files (migrations, view templates)
- required ENV variables added and described in `.env.example` and added to Heroku
- associated Heroku review app works correctly with introduced changes

### References
* [GitHub Issue #17](https://github.com/fs/rails-base-graphql-api/issues/17)
* [GitHub Pull Request #33](https://github.com/fs/rails-base-graphql-api/pull/33)
